### PR TITLE
DOC-2025: Document new `table_merge_content_on_paste` option

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-2025: Document new `table_merge_content_on_paste` option.

Changes:
* Added include statement pointing to new partial: /configuration/table_merge_content_on_paste.adoc.
* Added new partial: /configuration/table_merge_content_on_paste.adoc
* Documented new option, `table_merge_content_on_paste`, in this new file.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] Files has been included where required (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/configuration/table_merge_content_on_paste.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+When set to `+true+`, pasting a table into another table replaces the extant table with the pasted table. It merges the pasted table into the extant table, replacing the extant table in the process.
+
+When set to `+false+`, pasting a table into another table creates a new table within the selected cell of the extant table.
+
+The specific behavior, when this option is set to `+false+`, depends on the nature of the current selection.
+
+. If there is no selection (ie the insertion point is within a cell), pasting a table in
+  * pastes the table at the insertion point;
+  * places material in the cell before the insertion point above the pasted-in table; and
+  * places material in the cell after the insertion point after the pasted-in table.
```
Comment (brian-forte) [inline modules/ROOT/partials/configuration/table_merge_content_on_paste.adoc:17]:
```suggestion
  * places material in the cell after the insertion point below the pasted-in table.
```

**Review by TheSpyder — APPROVED** [review]
It could do with a "since TinyMCE 6.5" or perhaps "added in 6.5", which I believe we have templates for elsewhere in the docs

**Comment by brian-forte** [comment]
> It could do with a "since TinyMCE 6.5" or perhaps "added in 6.5", which I believe we have templates for elsewhere in the docs

To quote my commit message:

> DOC-2025: table_merge_content_on_paste.adoc
> 
> Added `include::` statement pointing to `/misc/admon-requires-6.5v.adoc`.

File: modules/ROOT/partials/configuration/table_merge_content_on_paste.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
+[[table_merge_content_on_paste]]
+== `+table_merge_content_on_paste+`
+
+include::partials$misc/admon-requires-6.5v.adoc[]
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/table_merge_content_on_paste.adoc:4]: @brian-forte Do we need to bump the version here to be `6.5.1`?

**Review by kemister85 — APPROVED** [review]

**Review by ArvinJ-H — APPROVED** [review]

File: modules/ROOT/partials/configuration/table_merge_content_on_paste.adoc (hunk 2 above)
Comment (MitchCTiny) [reply modules/ROOT/partials/configuration/table_merge_content_on_paste.adoc:4]: The new option would exist on 6.5 so shouldn't be necessary.

**Review by MitchCTiny — APPROVED** [review]

**Pull request merged**